src/main/kotlin/net/maizegenetics/phgv2/pathing/BuildKmerIndex.kt

```diff
@@ -138,6 +138,8 @@ class BuildKmerIndex: CliktCommand(help="Create a kmer index for a HaplotypeGrap
         val refRangeToIndexMap = graph.refRangeToIndexMap()
         val hapidToRefrangeMap = graph.hapIdToRefRangeMap()
 
+        val maxHapsToKeep = sampleGametes.size * 2
```

Comment:
The header comment at the top of BuildKmerIndex.kt says "Only hashes observed in exactly one reference range will be kept. ...".  WIll you update this comment to reflect the changes you are making here?